x/feegrant/types/basic_fee.go

```diff
@@ -20,11 +18,10 @@ var _ FeeAllowanceI = (*BasicFeeAllowance)(nil)
 // If remove is true (regardless of the error), the FeeAllowance will be deleted from storage
 // (eg. when it is used up). (See call to RevokeFeeAllowance in Keeper.UseGrantedFees)
 func (a *BasicFeeAllowance) Accept(ctx sdk.Context, fee sdk.Coins, _ []sdk.Msg) (bool, error) {
```

Comment:
could you update the method doc that mentions "timestamp/height"?
it has to be updated in other places in the code as well: types/fees.go and types/periodic_fee.go